src/ServicePulse.Host/vue/src/components/customchecks/CustomCheckView.vue

```diff
@@ -40,15 +31,15 @@ async function dismissCustomCheck(id: string) {
           </div>
         </div>
         <div>
-          <button type="button" class="btn btn-default" title="Dismiss this custom check so it doesn't show up as an alert" @click="dismissCustomCheck(customCheck.id)">Dismiss</button>
+          <button type="button" class="btn btn-default" title="Dismiss this custom check so it doesn't show up as an alert" @click="store.dismissCustomCheck(customCheck.id)">Dismiss</button>
         </div>
       </div>
     </div>
   </div>
 </template>
 
 <style scoped>
-@import "./list.css";
+@import "../list.css";
```

Comment:
```suggestion
@import "@/components/list.css";
```
Just for consistency?